pkg/collector/corechecks/cluster/ksm/customresources/vpa.go

```diff
@@ -21,7 +21,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/watch"
-	v1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1"
+	v1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2"
```

Comment:
Keeping the same alias but changing the underlying version was the smaller change, but happy to do the larger rename if people think it is worth it

Reply:
I think it should have been autoscalingv1` instead of just `v1`. I believe it's worth renaming, especially as it can be done automatically by IDE/Tools.

Reply:
`autoscalingv1` already exists, on line 19 (just out of view here)

Reply:
Right one is `autoscaling/v1` and the vpa is `autoscaling.k8s.io/v1`. I guess  `autoscalingk8sio` would not make a lot of sense so perhaps something like `vpav1` could make sense.

Reply:
Updated